### ISSUE TITLE
Allow querying an account without calling new_account in ACMEv2

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#6258]
If you do this with boulder, you get:
```
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 130, in query_registration
    return self._send_recv_regr(regr, messages.UpdateRegistration())
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 75, in _send_recv_regr
    response = self._post(regr.uri, body)
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 92, in _post
    return self.net.post(*args, **kwargs)
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 1109, in post
    return self._post_once(*args, **kwargs)
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 1123, in _post_once
    return self._check_response(response, content_type=content_type)
  File "/Users/bmw/Development/certbot/certbot/acme/acme/client.py", line 983, in _check_response
    raise messages.Error.from_json(jobj)
acme.messages.Error: urn:ietf:params:acme:error:malformed :: The request message was malformed :: No Key ID in JWS header
```
This is because [querying an account requires kid to be set](https://tools.ietf.org/html/draft-ietf-acme-acme-13#section-6.2), but if [self.net.account isn't set, kid isn't included](https://github.com/certbot/certbot/blob/c129ab2965f7ec4edf173b1cd900bd5add13cd2a/acme/acme/client.py#L938).

An easy fix for this would be to set `self.net.account = regr` in the beginning of `query_registration`, at least for ACMEv2. Doing it at the end of `query_registration` to store the account as given by the ACME server isn't a bad idea either.

This is useful for the scenario in #6246.

**Comment by bmw** [comment]
Fixed in #6251.

**Comment by evilaliv3** [comment]
@bmw @dovreshef: is there any easy fix that one can do in 0.25.1?

It would be great if you could suggest one as there are so many users that will continue to use suchversion given that is present in many debian/ubuntu versions.

Which is the correct code that a client that hasalready performed the registration should execute in order to renew it?
